packages/@webex/internal-plugin-metrics/src/rtcMetrics/index.ts

```diff
@@ -15,6 +15,22 @@ const parseJsonPayload = (payload: any[]): any | null => {
   }
 };
 
+interface IdType {
+  meetingId?: string;
+  callId?: string;
+}
+
+interface IMetricsAttributes {
+  type: string;
+  version: string;
+  userId: string;
+  correlationId: string;
+  connectionId: string;
```

Comment:
Would connectionId be applicable for calling cases?

Reply:
This value is local to rtcMetrics, not being passed from meeting also so, kept it as it is.